src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -2260,33 +2252,6 @@ struct X
                 );
         }
 
-        [Fact]
-        public void System_String__ConcatObject()
```

Comment:
We genuinely never emit this any more, so we can't trigger this diagnostic.

Reply:
Maybe please add the test back with whatever actual behavior it displays (no errors) verified, and also check that it does the right thing at runtime.

---
In reply to: [275796563](https://github.com/dotnet/roslyn/pull/35006#discussion_r275796563) [](ancestors = 275796563)